scripts/kconfig/kconfigfunctions.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) 2018-2019 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+# Types we support
+# 'string', 'int', 'hex', 'bool'
+conf_file = os.path.join(os.environ['PROJECT_BINARY_DIR'], "include/generated/generated_dts_board.conf")
+
+dt_defines = {}
+if os.path.isfile(conf_file):
+    with open(conf_file, 'r', encoding='utf-8') as fd:
+        line = fd.readline()
+        while line:
+            if '=' in line:
+                c = line.split('=')
```

Comment:
Another option is `key, val = line.split('=')`.

Reply:
not sure I follow how this change would work.

Reply:
`key, val = 'foo=bar'.split('=')` sets `key` to `foo` and `val` to `bar`. Python "unpacks" the RHS when you have multiple variables in the LHS.

Would make the code afterwards `dt_defines[key] = val.strip()`.

Reply:
Would get this along with the other change:

```python
dt_defines = {}
if os.path.isfile(conf_file):
    with open(conf_file, 'r', encoding='utf-8') as fd:
        for line in fd:
            if '=' in line:
                key, val = line.split('=')
                dt_defines[key] = val.strip()
```

I wonder if the `strip()` is needed.

Reply:
The strip() is needed, made this change.